regex-syntax/src/parser.rs

```diff
@@ -1047,19 +1058,19 @@ fn rev_concat(mut exprs: Vec<Expr>) -> Expr {
     }
 }
 
-// Returns ture iff the given character is allowed in a capture name.
+// Returns true if the given character is allowed in a capture name.
```

Comment:
Hah. The "iff" actually isn't a typo. It means, "if and only if." It's leftover shorthand from getting a math degree. :P